### ISSUE TITLE
Is it possible to adjust the initial globe position so it's not centered?

### ISSUE THREAD

**Issue by mbrodt** [vasturiano/react-globe.gl#3]
Hi - firstly, thank you so much for creating this library. I found it very easy to work with, despite being completely new to WebGL and 3D rendering in general.

I was wondering if it is possible to adjust the initial globe position so it is not centered. I can move it manually by adjusting the position of the underlying Three.js sphere, but then I would have to move all the points and atmosphere as well - so I was curious if there might be a better way?

Basically I need the canvas to take up the entire screen, but the globe should be more toward the bottom (vertically).

Thanks again for your work! :) 

Cheers

**Comment by vasturiano** [comment]
Thanks for reaching out @mbrodt!

For this, I would suggest to offset your canvas container element instead. You can make it occupy 80% of the screen height for example and align it on the bottom.
The very center of the canvas is set to be the center point of the globe, and it's not trivial to modify that.

**Comment by mbrodt** [comment]
Thanks for the quick response @vasturiano 

Yep, that's exactly what I have been doing so far. It does create this sort of weird "barrier" where the canvas ends and the normal background begins, so that's why I was curious if there was a way to adjust the positioning.
![image](https://user-images.githubusercontent.com/21239560/70329674-3b498600-183c-11ea-9107-bb7f8c09d4b2.png)

But it makes total sense now :) thanks for the explanation!

**Comment by IamMilev** [comment]
How can you offset the canvas so that weird barrier doesn't show up? @mbrodt

**Comment by gbreux** [comment]
I was looking for something similar as well and was doing the offset technique but if you offset the canvas too much outside the viewport (to only display a potion of a big globe for example and to prevent the "weird barrier"), the performance decrease a lot and the browser has a hard time.

 https://stripe.com is a good example: they position the canvas in a way that fits the viewport and doesn't position it outside.

**Comment by sspaeth1** [comment]
@gbreux   @IamMilev  access the THEE objects and offset.

    const world = Globe({})

            let worldScene = world.scene()
            worldScene.position.x = -1000

**Comment by AlbertMarashi** [comment]
> @mbrodt @gbreux @IamMilev access the THREE objects and offset.
> 
> ```
> const world = Globe({})
> 
>         let worldScene = world.scene()
>         worldScene.position.x = -1000
> ```

This causes the globe to start on the right, but then it orbits around to the left of the screen
